mmv1/products/logging/api.yaml

```diff
@@ -28,8 +28,7 @@ objects:
   - !ruby/object:Api::Resource
     name: "Metric"
     base_url: projects/{{project}}/metrics
-    # The % in self_link indicates that the name value should be URL-encoded.
-    self_link: "projects/{{project}}/metrics/{{%name}}"
```

Comment:
What is the purpose of this change? I thought it would not affect the provider, but it seems like it is causing some tests to fail

Reply:
I made this change because my local tests of terraform validator was failing, and I see that here in the Github are failing too. The documentation mentions that this test is the most important.
 But I observed too that with the change to use the `{{%name}}`, the terraform validator fails but the VCR-test is now passing.
 I'm investigating this.